cluster/manifests/zalando-iam-aws-proxy/deployment.yaml

```diff
@@ -0,0 +1,77 @@
+# {{ if eq .Cluster.Provider "zalando-eks" }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: zalando-iam-aws-proxy
+  namespace: kube-system
+  labels:
+    application: kubernetes
+    component: zalando-iam-aws-proxy
+spec:
+  selector:
+    matchLabels:
+      deployment: zalando-iam-aws-proxy
+  template:
+    metadata:
+      labels:
+        application: kubernetes
+        component: zalando-iam-aws-proxy
+        deployment: zalando-iam-aws-proxy
+      annotations:
+        logging/destination: "{{.Cluster.ConfigItems.log_destination_infra}}"
+    spec:
+      dnsConfig:
+        options:
+          - name: ndots
+            value: "1"
+      serviceAccountName: zalando-iam-aws-proxy
+      containers:
+      - name: proxy
+        image: mikkeloscar/zalando-iam-aws-proxy:2
```

Comment:
Needs to use a properly built image

Reply:
Done in https://github.com/zalando-incubator/kubernetes-on-aws/pull/7818/commits/98c4f8d315cc2b227ab985a3620c57bb8af58bdb